src/current/v25.2/create-policy.md

```diff
@@ -0,0 +1,92 @@
+---
+title: CREATE POLICY
+summary: The CREATE POLICY statement defines a new row-level security policy on a table.
+toc: true
+keywords: security, row level security, RLS
+docs_area: reference.sql
+---
+
+The `CREATE POLICY` statement defines a new [row-level security (RLS)]({% link {{ page.version.version }}/row-level-security.md %}) policy on a [table]({% link {{ page.version.version }}/schema-design-table.md %}).
+
+## Syntax
+
+<!--
+
+NB. This is commented out while we wait for a fix to DOC-12125
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/{{ page.release_info.crdb_branch_name }}/grammar_svg/create_policy_stmt.html %}
+</div>
+
+-->
+
+{% include_cached copy-clipboard.html %}
+~~~
+CREATE POLICY policy_name ON table_name
```

Comment:
It was a late change, but we added a `IF NOT EXISTS` option to this DDL:
https://github.com/cockroachdb/cockroach/issues/138835

Reply:
Nice, added it

Sidenote: we are having some issues with the SQL diagrams generation due to the machinery being spread across multiple repos, but i am planning to update this to reference the automatically generated diagram once I confirm that's working as expected. But will probably go with this for now for the cloud release to get it out the door